compass/ocean/tests/global_ocean/mesh/arm10to60/dynamic_adjustment/__init__.py

```diff
@@ -0,0 +1,240 @@
+from compass.ocean.tests.global_ocean.dynamic_adjustment import \
+    DynamicAdjustment
+from compass.ocean.tests.global_ocean.forward import ForwardStep
+
+
+class ARM10to60DynamicAdjustment(DynamicAdjustment):
+    """
+    A test case performing dynamic adjustment (dissipating fast-moving waves)
+    from an initial condition on the ARM10to60 MPAS-Ocean mesh
+
+    Attributes
+    ----------
+    restart_filenames : list of str
+        A list of restart files from each dynamic-adjustment step
+    """
+
+    def __init__(self, test_group, mesh, init, time_integrator):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : compass.ocean.tests.global_ocean.GlobalOcean
+            The global ocean test group that this test case belongs to
+
+        mesh : compass.ocean.tests.global_ocean.mesh.Mesh
+            The test case that produces the mesh for this run
+
+        init : compass.ocean.tests.global_ocean.init.Init
+            The test case that produces the initial condition for this run
+
+        time_integrator : {'split_explicit', 'RK4'}
+            The time integrator to use for the forward run
+        """
+        if time_integrator != 'split_explicit':
+            raise ValueError('{} dynamic adjustment not defined for {}'.format(
+                mesh.mesh_name, time_integrator))
+
+        restart_times = ['0001-01-01_06:00:00', '0001-01-01_12:00:00',
+                         '0001-01-02_00:00:00', '0001-01-03_00:00:00',
+                         '0001-01-04_00:00:00', '0001-01-07_00:00:00',
+                         '0001-01-31_00:00:00']
+        restart_filenames = [
+            'restarts/rst.{}.nc'.format(restart_time.replace(':', '.'))
+            for restart_time in restart_times]
+
+        super().__init__(test_group=test_group, mesh=mesh, init=init,
+                         time_integrator=time_integrator,
+                         restart_filenames=restart_filenames)
+
+        module = self.__module__
+
+        shared_options = \
+            {'config_AM_globalStats_enable': '.true.',
+             'config_AM_globalStats_compute_on_startup': '.true.',
+             'config_AM_globalStats_write_on_startup': '.true.',
+             'config_use_activeTracers_surface_restoring': '.true.'}
+
+        # first step
+        step_name = 'damped_adjustment_1'
+        step = ForwardStep(test_case=self, mesh=mesh, init=init,
+                           time_integrator=time_integrator, name=step_name,
+                           subdir=step_name)
+
+        namelist_options = {
+            'config_run_duration': "'00-00-00_06:00:00'",
+            'config_dt': "'00:00:30'",
+            'config_btr_dt': "'00:00:01.5'",
+            'config_Rayleigh_friction': '.true.',
+            'config_Rayleigh_damping_coeff': '1.0e-3'}
```

Comment:
For each step (except `simulation`), these are the values you'll want to adjust.  We could try the same as the WC14 to begin with but likely the time steps will need to be reduced because of the difference in minimum resolution.

Run each `damped_adjustment` step in compass on its own (rather than trying to run this as a suite).  Monitor the global CFL number and the kinetic energy.  If the CFL gets much over 0.1, you're in trouble.  If the KE spikes, you're in trouble.  Some combination of a shorter time step and more Rayleigh damping typically helps.